bin/expedius-install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+##----------------------------------------------------------------------
+## Expedius Install script
+##
+## Author: Dennis Warren @ Colcamex Resources
+## Date: February 2012
+## Updated: January 2020
+## This script is used to install Expedius Lab Auto Downloader onto
+## a server environment that supports Oscar EMR version 15*
+##----------------------------------------------------------------------
+
+# init variables
+TOMCAT_PATH=\/usr\/local\/tomcat\/webapps
+ADMIN_EMAIL=none
+APP_DATA=\/var\/lib\/expedius
+EMAIL_SERVICE=no
+EXPEDIUS_USERNAME=admin
+EXPEDIUS_PASSWORD=3xp3dius#1
+OSCAR_USERNAME=expedius
+OSCAR_PASSWORD=$(openssl rand -base64 8)
```

Comment:
Is this meant to be the actual oscar password? (for Oscar front-end?)

Reply:
The auto gen creates a random Oscar password and spits it out to the user at the end.  Not the ideal situation.  We should talk about how the process should work

Reply:
Yes. For the Oscar front end.